basic-fragments.tex

```diff
@@ -1,22 +1,15 @@
 \def\drivedefinition{
-{\bf drive} drive \# in dual drive disk units. \\
-The drive \# defaults to {\bf 0} and can be omitted on single drive units
-such as the 1541, 1571, or 1581.
+{\bf drive} is the drive \# in dual drive disk units. The drive \# defaults to {\bf 0} and can be omitted on single drive units such as the 1541, 1571, or 1581.

```

Comment:
Argument lists are currently styled to name the argument in bold, followed by a lowercase sentence fragment with an implied "is." IMO using initial-caps for the sentence fragment would be an improvement, and I wouldn't be opposed to that change globally. For now, I borrowed this from the 1st ed.

Reply:
You mean that the above should be written like:

{\bf drive} This is the drive \# in dual drive disk units. The drive \# defaults to {\bf 0} and can be ...

Reply:
To fully match the current style it should be left as is. The definitional sentence fragment that follows the term is lowercase. `{\bf drive} drive \# in dual drive disk units. \\ ...`

I would consider it an improvement to initial-caps the sentence fragment: `{\bf drive} Drive \# in dual drive disk units. \\ ...` But we'd need to do that consistently across the entire BASIC reference, which is many instances.

Reply:
I will revert these "is the" changes.